packages/playground/blueprints/public/blueprint-schema.json

```diff
@@ -1,1205 +1,1406 @@
+                  "type": "number"
+                },
+                "caption": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false
+            },
+            "step": {
+              "type": "string",
+              "const": "runPHP",
+              "description": "The step identifier."
+            },
+            "code": {
+              "type": "string",
+              "description": "The PHP code to run."
+            }
+          },
+          "required": [
+            "code",
+            "step"
+          ]
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "progress": {
+              "type": "object",
+              "properties": {
+                "weight": {
+                  "type": "number"
+                },
+                "caption": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false
+            },
+            "step": {
+              "type": "string",
+              "const": "runPHPWithOptions"
+            },
+            "options": {
+              "$ref": "#/definitions/PHPRunOptions",
+              "description": "Run options (See /wordpress-playground/api/universal/interface/PHPRunOptions)"
+            }
+          },
+          "required": [
+            "options",
+            "step"
+          ]
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "progress": {
+              "type": "object",
+              "properties": {
+                "weight": {
+                  "type": "number"
+                },
+                "caption": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false
+            },
+            "step": {
+              "type": "string",
+              "const": "runWpInstallationWizard"
+            },
+            "options": {
+              "$ref": "#/definitions/WordPressInstallationOptions"
+            }
+          },
+          "required": [
+            "options",
+            "step"
+          ]
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "progress": {
+              "type": "object",
+              "properties": {
+                "weight": {
+                  "type": "number"
+                },
+                "caption": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false
+            },
+            "step": {
+              "type": "string",
+              "const": "runSqlFile",
+              "description": "The step identifier."
+            },
+            "path": {
+              "type": "string",
+              "description": "The SQL file to run."
```

Comment:
there's ambiguity here between file paths on a local system and a URI/URL for paths online. can we clarify in the docs what "path" this path is? if we can include an example that could also speak loudly here.

Reply:
We're using `resource`s now, so the general documentation around providing data from any source should apply here.  How does this look? The format of the underlying data has been specified in the description:

https://github.com/WordPress/wordpress-playground/pull/745/files#diff-a2106348a53b7f8a5446c8879358be40539a2aee556c8525e79133f12076e53dR901

Reply:
@dmsnell  This file has been removed.

https://github.com/WordPress/wordpress-playground/pull/745/files